src/Engine.coffee

```diff
@@ -356,7 +356,7 @@ class Engine
       unless src.match(/gss/i)
         # Select a script from document that has gss in its src
         scripts = document.querySelectorAll('script[src*=gss]')[0]
-        if scripts.length
+        if scripts?.length
```

Comment:
Seems that this line never really worked properly. It was a fall back for the case when gss file is loaded asynchronously. I think we need to remove [0] here to make .length check pass